### DISCUSSION TITLE
Umi Max怎么启用面包屑

### DISCUSSION THREAD

**Discussion by Alextale777** [umijs/umi#12044] [General]
采用了umi Max版，但是不知道如何启用面包屑，同时不同页面的ProContainer也并未生效，不知道要启用啥配置

**Comment by fz6m** [comment]
max 里用的 layout 这个功能本质上是 pro components 的 `<ProLayout />` ，默认情况下面包屑会根据你配置的路由层级 name 自动生成，如果需要自定义或修改 layout 相关的行为，参考 [ProLayout 的配置项](https://procomponents.ant.design/components/layout#prolayout) ，配到如下位置即可：

```ts
// src/app.tsx

export const layout = () => {
  return {
    // ProLayout 的配置项
    // ...
  }
}
```

**Reply by hcy1996** [reply]
同遇到这个问题，按道理默认就有面包屑的吧，但是确实没看到

**Comment by yimingyu001** [comment]
这个问题解决了吗， 我这个也遇到了同样的问题
